### DISCUSSION TITLE
GSoC 2022: Support for simulated version of java.security.Keystore APIs

### DISCUSSION THREAD

**Discussion by spydyboy** [robolectric/robolectric#7141] [General]
Hi everyone,
Greetings!
I want to explore this project Support for simulated version of Keystore APIs as part of GSoC program.
As a starter, have experience with TDD Java development.
Any tips/suggestion for the starting point of this project.
Thanks

**Comment by utzcoz** [comment]
@spydyboy Welcome to Robolectric, and it's glad that you have interested in project to improve simulation for `KeyStore` APIs. It's recommended to follow http://robolectric.org/contributing/ to setup Robolectric development environment locally firstly. Feel free to ask any question about Robolectric and this project. Hi @hoisie, could you help to explain more about current status of this project? Thanks.

**Comment by hoisie** [comment]
Hi @spydyboy 

I would be happy to provide some guidance for this project. I was thinking to start we could look into Conscrypt support, since I don't think that would be too hard.

As a first step it would be great to set up a development environment where you can build Robolectric. Could you try to follow the instructions at http://robolectric.org/building-robolectric/ and see if you can get to the point where you can build Robolectric? What OS are you using?
